aws/network-primitives/scripts/installer.sh

```diff
@@ -3,7 +3,7 @@
 # updates
 export DEBIAN_FRONTEND=noninteractive
 sudo apt update -y
-sudo apt install -y curl
+sudo apt install curl gnupg openssl net-tools -y
```

Comment:
are you sure you need all of there packages ?

Reply:
Yes, i need to update to latest curl (used to get external IP and install docker) gnupg (update) for apt keyring, latest openssl and net-tools to check ports with netstat.